rasa/utils/endpoints.py

```diff
@@ -144,12 +144,15 @@ def combine_parameters(
                 headers=headers,
                 params=self.combine_parameters(kwargs),
                 **kwargs,
-            ) as resp:
-                if resp.status >= 400:
+            ) as response:
+                if response.status >= 400:
                     raise ClientResponseError(
-                        resp.status, resp.reason, await resp.content.read()
+                        response.status, response.reason, await response.content.read()
                     )
-                return await getattr(resp, return_method)()
+                try:
+                    return response.json
```

Comment:
shouldn't it be `await response.json()`?

Reply:
I tried this but it actually yells, there's something weird bc it is a regular property that has a coroutine to get it

Reply:
nope, now it's working. don't know why that was weird before.